ports/node-api/portfile.cmake

```
@@ -0,0 +1,79 @@
+set(NODEJS_VERSION 18.12.1)

```

```suggestion
vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
```

You could use `${VERSION}` directly in the portfile.cmake instead of set a new version variable.

---

Added. Thank you